homeassistant/components/netatmo/webhook.py

```diff
@@ -0,0 +1,64 @@
+"""The Netatmo integration."""
+import logging
+
+from .const import (
+    ATTR_EVENT_TYPE,
+    ATTR_FACE_URL,
+    ATTR_ID,
+    ATTR_IS_KNOWN,
+    ATTR_NAME,
+    ATTR_PERSONS,
+    DATA_PERSONS,
+    DEFAULT_PERSON,
+    DOMAIN,
+    NETATMO_EVENT,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def handle_webhook(hass, webhook_id, request):
+    """Handle webhook callback."""
+    try:
+        data = await request.json()
+    except ValueError:
+        return None
+
+    _LOGGER.debug("Got webhook data: %s", data)
+
+    event_type = data.get(ATTR_EVENT_TYPE)
+
+    if event_type == "outdoor":
```

Comment:
Is outdoor a special event that has multiple events embedded?

Reply:
Yes, exactly that. 

https://dev.netatmo.com/apidocumentation/security#events

> Outdoor Subtypes
> For «Outdoor» events, an «event_list» array is available and provides more details about the event detected by the Smart Outdoor Camera. If several events are detected in the same video (e.g. a car is seen and people come out of it), there will only be one «Outdoor» event and its «eventlist» array will contain all of the events detected. Events are aggregated by video.